pkg/fileutil/fileutil.go

```diff
@@ -167,6 +168,11 @@ func CopyTree(src, dest string, uidRange *uid.UidRange) error {
 		return nil
 	}
 
+	fis, err := ioutil.ReadDir(dest)
```

Comment:
Nope, it will try to Mkdir the target directory, which already exists, and fail. That's why I'm removing it if it exists and it's empty.

And yes, we must document the function.